src/main/java/com/invicto/streaming_platform/persistence/repository/UserRepository.java

```diff
@@ -1,11 +1,11 @@
-package com.invicto.streamingPlatform.persistence.repository;
+package com.invicto.streaming_platform.persistence.repository;
 
-import com.invicto.streamingPlatform.persistence.model.User;
+import com.invicto.streaming_platform.persistence.model.User;
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.Optional;
 
 public interface UserRepository extends CrudRepository<User, Long> {
 	Optional<User> findByLogin(String login);
-	Optional<User> findByEmailAddress(String emailAddress);
+	Optional<User> findByEmail(String emailAddress);
```

Comment:
Replace "emailAdress" to "email" in params too?
`Optional<User> findByEmail(String email);`